tony-history-server/app/controllers/JobConfigPageController.java

```diff
@@ -37,7 +37,7 @@ public Result index(String jobId) {
     }
 
     List<JobConfig> listOfConfigs;
-    String tonyHistoryFolder = config.getString("tony.historyFolder");
+    String tonyHistoryFolder = config.getString("tony.history.location");
```

Comment:
You're planning to replace with `TonyConfigurationKeys.TONY_HISTORY_LOCATION` in a follow-up PR?

Reply:
Yeah, we can discuss this in the sync-up tmr.